js/dtos/MeteorDto.js

```diff
@@ -0,0 +1,27 @@
+class MeteorDto {
+  constructor(data) {
+    this.startDate = specifyStartDate(data)
+    this.endDate = specifyEndDate(data)
+    this.count = data.count
+    this.wereDangerous = specifyWereDangerous(data)
+  }
+}
+
+const specifyStartDate = (data) => {
+  const startDate = data.startDate
+  const endDate = data.endDate
+  return startDate === undefined && endDate !== undefined ? endDate : startDate
+}
+
+const specifyEndDate = (data) => {
+  const startDate = data.startDate
+  const endDate = data.endDate
+  return endDate === undefined && startDate !== undefined ? startDate : endDate
+}
+
+const specifyWereDangerous = (data) => {
+  const wereDangerous = data.wereDangerous
+  return wereDangerous !== undefined && wereDangerous.toLowerCase() === 'true'
```

Comment:
It's enough to check just `wereDangerous.toLowerCase() === 'true'`

Reply:
Your suggestion is good, because meteorDto is filled with request after validation. 
But if I build this object not from request, then "underfined check" will be required. In tests for example.
What do you think?

Reply:
Actually it won't, if you skip .toLowerCase() call and will perform validation before passing data to dto. Then you'll be sure that if request contains `wereDangerous` field, it is already correct value - 'true' or 'false', so your function will work pretty good (undefined !== true)